server/gabble/config/env.go

```diff
@@ -0,0 +1,61 @@
+package config
```

Comment:
Resolves Gable configurations for `.env` files.
If the user wants to use either `toml`, `yml` or `json` as a config file, a "handler" must be implemented.
The configuration is independent from its provider.